comp/otelcol/collector-contrib/impl/collectorcontrib.go

```diff
@@ -343,13 +381,51 @@ func (c *collectorcontribImpl) OTelComponentFactories() (otelcol.Factories, erro
 	}
 
 	exporterList := []exporter.Factory{
-		// datadogexporter.NewFactory(),
```

Comment:
Should we remove this? It's already mentioned as temporarily disabled above.

Reply:
Yes it should be removed. Adding it as a dependency pulls in most of the agent code, which is bad. Instead, there is a narrowing datadogexporter coming down the pipeline that will eventually be used instead.